docs/architecture/application/cli.md

```diff
@@ -519,11 +521,27 @@ product_blocks:
         description: "ID of the node in the inventory management system"
         type: int
         required: active
+      - name: under_maintenance
+        description: "node is under maintenance"
+        type: bool
+        required: initial
+        default: False
+
+workflows:
+  - name: terminate
+    validations:
+      - id: can_only_terminate_when_under_maintenance
+        description: "Can only terminate when the node is placed under maintenance"
+  - name: validate
+    enabled: false
+    validations:
+      - id: validate_ims_administration
```

Comment:
SURF specific?

Reply:
IMS refers to the more generic term Inventory Managent System, not to a specific product with that same name.